src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -439,4 +461,14 @@ export default {
 	background-color: var(--color-background-hover);
 	border-radius: 8px;
 }
+
+.highlight-animation {
+	animation: highlight-animation 2s 1;
+}
+
+// TODO: define these colors globally (these are borrowed from the files app upload.css)
+@keyframes highlight-animation {
+	0% { background-color: rgba(255, 255, 140, 1); }
```

Comment:
Need to introduce scss vars in server so it's themed somehow and dark aware

Reply:
I now simply switched to the existing hover variable